//multiple exception in java



public class Main {
	public static void main(String[] args) {
		int i=2;
		int j=0;
		int num[]=new int[5];
		String str= null;
		try{
		    j=18/i;
		 //   System.out.println(num[5]);
		    System.out.println(str.length());
		}
		catch(ArithmeticException e){
		        System.out.println("the num should be greaterthan 0");
		}
		catch(ArrayIndexOutOfBoundsException e){
		        System.out.println("the array has no space");
		}
		catch(Exception e){
		        System.out.println("something went wrong");
		}
		}
}
